### PR TITLE
Generic Parsing: Added list_to_value_dict

### PR DESCRIPTION
**Pull request by Neodia**

### CODE REVIEW

File: dags/common/parsing/generic_parsing.py [hunk 1]
```diff
@@ -3,3 +3,7 @@ def take_first(arr):
         return next(filter(None, arr))
     except StopIteration:
         return None
+
+
+def list_to_value_dict(arr, key="value"):
+    return [{key: val} for val in arr]
```
Comment (drjova) [inline dags/common/parsing/generic_parsing.py]: Should we check `if val`? in case the value is empty or None?

Comment (Neodia) [reply dags/common/parsing/generic_parsing.py]: Do we want {key: None} in our output ?

Comment (drjova) [reply dags/common/parsing/generic_parsing.py]: No, we are anyway removing them https://github.com/SCOAP3/scoap3-next/blob/master/scoap3/modules/workflows/workflows/articles_upload.py#L473 I would say to ignore empty and None values

Comment (Neodia) [reply dags/common/parsing/generic_parsing.py]: Added check and test 👍

**Review by drjova — APPROVED** [review]
LGTM, just the fix for the ref in the commit message.

**Pull request merged**